packages/salesforcedx-vscode-core/test/jest/context/workspaceContext.test.ts

```diff
@@ -0,0 +1,9 @@
+describe('workspaceContext', () => {
+  describe('handleCliConfigChange', () => {
+    it('should update context variables and UI elements when the config file changes', () => {});
```

Comment:
These are coming in a follow up PR?

Reply:
I'm hoping to get them in this one. Working on them now.